gluten-core/src/main/scala/org/apache/spark/sql/execution/ui/GlutenUiUtils.scala

```diff
@@ -17,13 +17,12 @@
 package org.apache.spark.sql.execution.ui
 
 import org.apache.gluten.config.GlutenConfig
-import org.apache.gluten.config.GlutenConfig.GLUTEN_UI_ENABLED
 import org.apache.gluten.events.GlutenEvent
 
 import org.apache.spark.SparkContext
 import org.apache.spark.status.ElementTrackingStore
 
-object GlutenUIUtils {
+object GlutenUiUtils {
```

Comment:
nit: UI is the abbreviation of User Interface. Is it more appropriate to keep the status quo? The same is true for spark, such as `org.apache.spark.ui.UIUtils`.